galsim/table.py

```diff
@@ -195,20 +195,20 @@ def __call__(self, x):
             if dimen > 2:
                 raise ValueError("Arrays with dimension larger than 2 not allowed!")
             elif dimen == 2:
-                f = np.empty_like(x.ravel()).astype(float)
-                self.table.interpMany(x.astype(float).ravel(),f)
+                f = np.empty_like(x.ravel(), dtype=float)
```

Comment:
I just updated my test case to use 2d arrays (I was using 1d) and the improvement shrank from 60% to 20% on my system.  So for most realistic use cases, the improvement won't be that much.  But I suspect that there are quite a few places where we could take advantage of this.